static/app/views/alerts/wizard/options.tsx

```diff
@@ -40,33 +45,44 @@ export const AlertWizardAlertNames: Record<AlertType, string> = {
   fid: t('First Input Delay'),
   cls: t('Cumulative Layout Shift'),
   custom: t('Custom Metric'),
+  crash_free_sessions: t('Crash Free Session Rate'),
+  crash_free_users: t('Crash Free User Rate'),
 };
 
-export const AlertWizardOptions: {
-  categoryHeading: string;
-  options: AlertType[];
-}[] = [
-  {
-    categoryHeading: t('Errors'),
-    options: ['issues', 'num_errors', 'users_experiencing_errors'],
-  },
-  {
-    categoryHeading: t('Performance'),
-    options: [
-      'throughput',
-      'trans_duration',
-      'apdex',
-      'failure_rate',
-      'lcp',
-      'fid',
-      'cls',
-    ],
-  },
-  {
-    categoryHeading: t('Other'),
-    options: ['custom'],
-  },
-];
+type AlertWizardCategory = {categoryHeading: string; options: AlertType[]};
+export function alertWizardCategories(org: Organization): AlertWizardCategory[] {
+  const options: AlertWizardCategory[] = [
+    {
+      categoryHeading: t('Errors'),
+      options: ['issues', 'num_errors', 'users_experiencing_errors'],
+    },
+    {
+      categoryHeading: t('Performance'),
+      options: [
+        'throughput',
+        'trans_duration',
+        'apdex',
+        'failure_rate',
+        'lcp',
+        'fid',
+        'cls',
+      ],
+    },
+    {
+      categoryHeading: t('Other'),
+      options: ['custom'],
+    },
+  ];
+
+  if (org.features.includes('crash-rate-alerts')) {
+    options.splice(1, 0, {
+      categoryHeading: t('Sessions'),
+      options: ['crash_free_sessions', 'crash_free_users'],
+    });
+  }
```

Comment:
Would this be easier to read if it was part of the `options` declaration? That way you'd be able to see the intended ordering instead of having to parse the `splice()` call.

Reply:
I agree, splice is never easy to parse.